packages/dd-trace/src/opentelemetry/context_manager.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const { AsyncLocalStorage } = require('async_hooks')
+const { trace, ROOT_CONTEXT } = require('@opentelemetry/api')
+
+const SpanContext = require('./span_context')
+const tracer = require('../../')
+
+// Horrible hack to acquire the otherwise inaccessible SPAN_KEY so we can redirect it...
```

Comment:
Can you explain in the comment why the workaround is needed?

Reply:
Also, what is a span key?

Reply:
Is it known by OpenTelemetry that this is needed when implementing the API? It seems like a pretty serious oversight.